src/Mobs/Wolf.cpp

```diff
@@ -29,7 +29,7 @@ cWolf::cWolf(void) :
 
 bool cWolf::DoTakeDamage(TakeDamageInfo & a_TDI)
 {
-	if (super::DoTakeDamage(a_TDI))
+	if (!super::DoTakeDamage(a_TDI))
```

Comment:
Regardless of this change, the `!` should have been there previously. Is there a reason it's not, or is it just a bug?